### DISCUSSION TITLE
Copilot in mobile app doesn’t understand its context

### DISCUSSION THREAD

**Discussion by kyelewis** [community/community#141720] [Copilot Conversations]
Even if asked/reminded that it is in the mobile app, asking copilot questions about app functions often provides results about the desktop website without context.

For example, asking it how to create a repo in the app returned instructions for a button that does not exist in the app, and asking it to clarify, it forgets that i asked about the mobile app (but it should already understand that context anyway)

**Comment by bronglil** [comment]
It seems that GitHub Copilot is not retaining the context when you specify you're using the mobile app, often providing desktop-related instructions instead. To improve this, Copilot should enhance its ability to understand and maintain the platform context, ensuring that responses are relevant to the mobile app. Additionally, introducing prompts for platform clarification and mobile-specific guides could help provide more accurate and helpful instructions. If this issue persists, I recommend submitting feedback directly through the app to assist in refining Copilot's performance in the mobile environment.
